### PR TITLE
Optimise images from Contentful

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: plugins/vue-filters.js [hunk 1]
```diff
@@ -16,3 +16,10 @@ Vue.filter('truncate', (val, char, ellipsis) => {
   return val.length > char ? val.substring(0, char) + ellipsis : val;
 });
 
+Vue.filter('optimisedImageUrl', (imageUrl) => {
+  const hostname = imageUrl.match(/\/\/([^/]+)\//)[1];
+  if (hostname === 'images.ctfassets.net') {
+    return imageUrl + '?fm=jpg&fl=progressive&q=25';
```
Comment (lbiedinger) [inline plugins/vue-filters.js]: q=25 will be too much, it results in quite noticeable artifacts.

Comment (rwd) [reply plugins/vue-filters.js]: Increased to 50.

File: components/browse/BrowseContentCard.vue [hunk 2]
```diff
@@ -35,12 +39,15 @@
           } else {
```
Comment (monkemedia) [inline components/browse/BrowseContentCard.vue]: Don't need an else here as we have a return above it

File: components/browse/BrowseContentCard.vue [hunk 3]
```diff
@@ -35,12 +39,15 @@
...
             return this.fields.image;
           }
-        } else if (this.fields.image && this.fields.image.fields && this.fields.image.fields.file) {
+        } else if (this.imageIsContentfulAsset) {
           return this.fields.image.fields.file.url;
         } else {
```
Comment (monkemedia) [inline components/browse/BrowseContentCard.vue]: Same here

File: pages/index.vue [hunk 4]
```diff
@@ -4,7 +4,8 @@
   >
     <HeroBanner
       v-if="page.primaryImageOfPage"
-      :hero-image="page.primaryImageOfPage.fields.image.fields.file.url"
+      :image-url="page.primaryImageOfPage.fields.image.fields.file.url"
+      :image-content-type="page.primaryImageOfPage.fields.image.fields.file.contentType"
       :headline="page.primaryImageOfPage.fields.headline"
       :description="page.primaryImageOfPage.fields.description"
       :identifier="page.primaryImageOfPage.fields.identifier"
```
Comment (monkemedia) [inline pages/index.vue]: I wonder if it would make better reading if we used computed properties here? What are your thoughts?

Comment (rwd) [reply pages/index.vue]: Good thinking.

**Pull request merged**